carma/launch/ros2_rosbag.launch.py

```diff
@@ -65,7 +65,10 @@ def record_ros2_rosbag(context: LaunchContext, vehicle_config_param_file, rosbag
                             exclude_topics_regex += str(topic) + "|"
 
                 proc = ExecuteProcess(
-                        cmd=['ros2', 'bag', 'record', '-s', 'mcap', '--qos-profile-overrides-path', overriding_qos_profiles, '-o', '/opt/carma/logs/rosbag2_' + str(datetime.now().strftime('%Y-%m-%d_%H%M%S')), '-a', '-x', exclude_topics_regex],
+                        cmd=['ros2', 'bag', 'record', '-a', '-s', 'mcap',
```

Comment:
This doesn't seem to actually have any change. Can we make it have no change for github blame cleanliness?

Reply:
I actually did reformat this to be multiple lines in order to make it easier to read

Reply:
ah okay yeah